### PR TITLE
Add a story teller app

### PR DESCRIPTION
**Pull request by ErikCH**
*Description of changes:*
This PR adds a story teller app as another example to show on how to use Amplify AI kit.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by atierian — APPROVED** [review]
Reviewed the backend part. Looks great, thanks for adding this!

**Review by dbanksdesign — APPROVED** [review]
LGTM!

File: story-teller/amplify/backend.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { defineBackend } from "@aws-amplify/backend";
+import { auth } from "./auth/resource";
+import { data } from "./data/resource";
+import {
+  generateImage,
+  MODEL_ID as MODEL_ID_STABLE,
```
[the remark this answers is not in the document]
Comment (ErikCH) [reply story-teller/amplify/backend.ts]: Updated

File: story-teller/amplify/backend.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+} from "./functions/generateImage/resource";
+import { Effect, PolicyStatement } from "aws-cdk-lib/aws-iam";
+import { storage, knowledgeBaseBucket } from "./storage/resource";
+import { getNews } from "./functions/getNews/resource";
+
+const KB_REGION = "us-west-2";
+
+// Replace with your own Knowledge Base ID
```
[the remark this answers is not in the document]
Comment (ErikCH) [reply story-teller/amplify/backend.ts:11]: added

**Review by dbanksdesign — APPROVED** [review]

**Pull request merged**